app/scripts/metamask-controller.js

```diff
@@ -3191,6 +3196,7 @@ export default class MetamaskController extends EventEmitter {
         txController.updatePreviousGasParams.bind(txController),
       abortTransactionSigning:
         txController.abortTransactionSigning.bind(txController),
+      getLayer1GasFee: txController.getLayer1GasFee.bind(txController),
```

Comment:
Concerning the comments about `networkClientId` ideally being optional.

Pending a new controller release, could we instead make it optional here by populating it within an arrow function if not provided?

We could read it directly from the `NetworkController` state as we currently do above?

Reply:
Not required as the controller signature has been updated.